qiskit/circuit/library/standard_gates/x.py

```diff
@@ -578,7 +587,10 @@ def qasm(self):
         old_name = self.name
         self.name = "c3sqrtx"
         try:
-            return super().qasm()
+            # pylint: disable=cyclic-import
+            from qiskit.circuit.quantumcircuit import _instruction_qasm2
+
+            return _instruction_qasm2(self)
         finally:
             self.name = old_name
```

Comment:
I think it's fine for this to continue to call up to `super().qasm()` - we're already in a deprecated context.

Reply:
Huh? I think I only put that in because I was forced to (by the test suite, I thin). I recheck it. And revert if possible.

Reply:
Done in a790c38